chat-plugins/info.js

```diff
@@ -531,8 +531,12 @@ var commands = exports.commands = {
 						if (!template.learnset) template = Tools.getTemplate(template.baseSpecies);
 						if (!template.learnset) continue;
 						var fullLearnset = template.learnset;
-						while (template.prevo) {
-							template = Tools.getTemplate(template.prevo);
+						while (template.prevo || (template.eventPokemon && template.baseSpecies !== template.species)) {
+							if (template.prevo) {
+								template = Tools.getTemplate(template.prevo);
+							} else {
+								template = Tools.getTemplate(template.baseSpecies);
+							}
 							for (var move in template.learnset) {
 								if (!fullLearnset[move]) fullLearnset[move] = template.learnset[move];
```

Comment:
Also this needs to be moved out of `chat-plugins/info.js` and into `tools.js` or something. Modifications to core data files should not be done by commands.